tools/ci_connector_ops/ci_connector_ops/pipelines/README.md

```diff
@@ -0,0 +1,81 @@
+# POC of CI connector pipelines in python
+
+This Python subpackage of `ci-connector-ops` gathers the POC code we're working on to:
+- Rewrite [airbyte-python.gradle](https://github.com/airbytehq/airbyte/blob/7d7e48b2a342a328fa74c6fd11a9268e1dcdcd64/buildSrc/src/main/groovy/airbyte-python.gradle) and [airbyte-connector-acceptance-test.gradle](https://github.com/airbytehq/airbyte/blob/master/buildSrc/src/main/groovy/airbyte-connector-acceptance-test.gradle) in Python.
+- Centralize the CI logic for connector testing 
+- Try out Dagger.io as a promising tool that can provide parallelism and caching out of the box for CI 
+
+## Install and use
+From `airbyte` repo root:
+
+### Install
+```bash
+cd tools/ci_connector_ops
+python -m venv .venv (please use Python 3.10)
```

Comment:
How do we enforce or change our python version?

Reply:
The package declares that it requires Python 3.10 here: https://github.com/airbytehq/airbyte/blob/1c49dc760ef31fd5c5675a1557fc42f6ba1b56e9/tools/ci_connector_ops/setup.py#L51

I'm using pyenv to switch my python version to 3.10 when I'm working on this project. Our developers should have pyenv installed as I think it's part of our core python development tooling.

This file says to pyenv to switch to 3.10 when you're working on this code directory:
https://github.com/airbytehq/airbyte/blob/1c49dc760ef31fd5c5675a1557fc42f6ba1b56e9/tools/ci_connector_ops/.python-version#L1